test/integration/targets/filters/tasks/main.yml

```
@@ -114,7 +114,7 @@
       - "'local' == ['localhost']|map('extract',hostvars,'ansible_connection')|list|first"
       - "'local' == ['localhost']|map('extract',hostvars,['ansible_connection'])|list|first"
   # map was added to jinja2 in version 2.7
-  when: "{{ ( lookup('pipe', '{{ ansible_python[\"executable\"] }} -c \"import jinja2; print(jinja2.__version__)\"')  is version('2.7', '>=') ) }}"
+  when: "( lookup('pipe', 'ansible_python[executable] -c \"import jinja2; print(jinja2.__version__)\"')  is version('2.7', '>=') )"
```

This won't work. I think what you want is instead, also with some other cleanups:

The important part is not having `ansible_python.executable` within the quotes, and fixing the key/attribute lookup.

```suggestion
  when: lookup('pipe', ansible_python.executable ~ ' -c "import jinja2; print(jinja2.__version__)"') is version('2.7', '>=')
```

---

Thank you!